client/store/users.js

```diff
@@ -72,7 +69,8 @@ function singleUserReducer(user = {}, action) {
 
 const rootReducer = combineReducers({
   allUsers: allUsersReducer,
-  singleUser: singleUserReducer,
+  //singleUser: singleUserReducer,
+  singleUserReducer,
```

Comment:
The logged in user's projects show up in redux as state.users.singleUserReducer.projects.   Need projects to populate onto projects page.  Question: why do we need to get the whole user object with the projects instead of just getting the projects based on what user is logged in?